clifford/_multivector.py

```diff
@@ -901,9 +852,9 @@ def basis(self) -> List['MultiVector']:
         return thisBasis
 
     def join(self, other) -> 'MultiVector':
-        """Returns the join of two blades.
-              .
-        J = A ^ B
+        r"""The join of two blades.
+
+        :math:`J = A \wedge B`
```

Comment:
Is there a difference in notation between the join and wedge?

Reply:
I don't know of any good notation to differentiate between them, maybe some stuff in the end of Leo's book where he talks about them and delta product etc?